modules/contact-form/grunion-contact-form.php

```diff
@@ -2217,6 +2218,10 @@ function __construct( $attributes, $content = null, $form = null ) {
 		// parse out comma-separated options list (for selects, radios, and checkbox-multiples)
 		if ( !empty( $attributes['options'] ) && is_string( $attributes['options'] ) ) {
 			$attributes['options'] = array_map( 'trim', explode( ',', $attributes['options'] ) );
+
+            if( !empty( $attributes['values'] ) && is_string( $attributes['values'] ) ) {
```

Comment:
Could you add the necessary spaces around brackets to respect WordPress' coding standards?
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage